hidimstat/test/test_LOCO.py

```diff
@@ -0,0 +1,48 @@
+import numpy as np
+from sklearn.linear_model import LinearRegression
+from sklearn.model_selection import train_test_split
+
+from hidimstat.LOCO import LOCO
+
+
+def test_LOCO(linear_scenario):
```

Comment:
You have to use snale_case or CamelCase, but not both in the same string. -> `test_loco`